### PR TITLE
Feature merchant reporting

### PR DESCRIPTION
**Pull request by eoliveros**
new pr for customer reporting and the changes to the test_api for a new function that couldnt get it to work.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -66,6 +66,7 @@ def check_auth(api_key_token, nonce, sig, body):
     if not api_key:
         return False, "not found", None
     res, reason = check_hmac_auth(api_key, nonce, sig, body)
+    print('check auth reason: '+str(reason))
```
Comment (djpnewton) [inline app.py]: remove

File: app.py [hunk 2]
```diff
@@ -254,6 +274,7 @@ def claim():
         port = int(os.environ.get("PORT", 5000))
         print("binding to port: %d" % port)
         socketio.run(app, host="0.0.0.0", port=port)
+
```
Comment (djpnewton) [inline app.py]: remove extra space

**Comment by djpnewton** [comment]
I pushed a commit (https://github.com/zap-me/zapm_server/pull/7/commits/02802928cf8f836295660596195bfc85989d9f39) that fixes most of the problems

take a look, there was some typos and stuff

File: app.py (hunk 1 above)
Comment (eoliveros) [reply app.py]: Removed this line.

File: app.py (hunk 2 above)
Comment (eoliveros) [reply app.py]: Removed the space.

**Comment by eoliveros** [comment]
closing...

**Pull request closed without merging**